### PR TITLE
[DOCS] Add table listing supported evaluation parameter expressions to the docs

### PR DESCRIPTION
**Pull request by talagluck**
Adds a table containing the list of supported Evaluation Parameter expressions to the Evaluation Parameters docs page.

- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Appropriate tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/terms/evaluation_parameter.md [hunk 1]
```diff
@@ -154,3 +154,18 @@ This Expectation will fail (the NYC taxi data allows for four types of payments)
 - You cannot currently combine complex values with arithmetic expressions.
 :::
 
+The complete list of expressions usable in Evaluation Parameters and their corresponding functions is below:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/terms/evaluation_parameter.md]:
```suggestion
The expressions and the corresponding functions that you can use in Evaluation Parameters are listed in the following table:
```
image

File: docs/docusaurus/docs/terms/evaluation_parameter.md [hunk 2]
```diff
@@ -154,3 +154,18 @@ This Expectation will fail (the NYC taxi data allows for four types of payments)
...
+
+| **Evaluation Parameter Expression** | **Python Function Call**                                      |
```
Comment (kwcanuck) [inline docs/docusaurus/docs/terms/evaluation_parameter.md]: Confirm both column headings have the same justification - left or center.

**Review by kwcanuck — APPROVED** [review]
There is a typo outside of the edit:

- line 98 - arithmatic

**Review by Rachel-Reverie — APPROVED** [review]
LGTM

**Pull request merged**